getting-started-docker.txt

```diff
@@ -0,0 +1,32 @@
+# Developing using docker
```

Comment:
Should this be a markdown file, and should it be part of the main README.md?

Reply:
You know what I completely forgotten I had done this, I will add this in at bottom of the main README.md